include/revng/Model/TypePathHelpers.h

```diff
@@ -59,4 +59,14 @@ inline std::string path(const model::Segment &Segment) {
   return "/Segments/" + key(Segment);
 }
 
+inline std::string variablePath(const model::Function &Function,
+                                const model::LocalIdentifier &Variable) {
+  return path(Function) + "/LocalVariables/" + key(Variable);
```

Comment:
Did we add doc for these new locations to the doc?

Reply:
Done, please take a look at the dedicated commit near the end.